doc/src/fix_ave_moments.rst

```diff
@@ -0,0 +1,286 @@
+.. index:: fix ave/moments
+
+fix ave/moments command
+=======================
+
+Syntax
+""""""
+
+.. code-block:: LAMMPS
+
+   fix ID group-ID ave/moments Nevery Nrepeat Nfreq value1 value2 ... moment1 moment2 ... keyword args ...
+
+* ID, group-ID are documented in :doc:`fix <fix>` command
+* ave/moments = style name of this fix command
+* Nevery = use input values every this many time steps
+* Nrepeat = # of times to use input values for calculating averages
+* Nfreq = calculate averages every this many time steps
+* one or more input variables can be listed
+* value = v_name
+
+  .. parsed-literal::
+
+       c_ID = global scalar calculated by a compute with ID
+       c_ID[I] = Ith component of global vector calculated by a compute with ID, I can include wildcard (see below)
+       f_ID = global scalar calculated by a fix with ID
+       f_ID[I] = Ith component of global vector calculated by a fix with ID, I can include wildcard (see below)
+       v_name = value calculated by an equal-style variable with name
+       v_name[I] = value calculated by a vector-style variable with name, I can include wildcard (see below)
+
+* one or more moments to compute can be listed
+* moment = *mean* or *stddev* or *variance* or *skew* or *kurtosis*, see exact definitions below.
+* zero or more keyword/arg pairs may be appended
+* keyword = *start* or *delay*
+
+  .. parsed-literal::
+
+       *start* args = Nstart
+         Nstart = invoke first after this time step
+       *history* args = Ndelay
+         Ndelay = keep a history of up to Ndelay invocations
+
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   fix 1 all ave/moments 1 1000 100 v_volume mean sttdev
+   fix 1 all ave/moments 1 200 1000 v_volume variance kurtosis history 10
+
+Description
+"""""""""""
+
+.. versionadded:: TBD
+
+Using one or more variables as input every few time steps, calculate the
+moments of the underlying distribution based on the samples collected
+over a time step window. The definitions of the moments calculated are
+given below.
+
+The group specified with this command is ignored.  However, note that
+specified values may represent calculations performed by computes and
+fixes which store their own "group" definitions.
+
+Each listed value can be the result of a :doc:`compute <compute>` or
+:doc:`fix <fix>` or the evaluation of an equal-style or vector-style
+:doc:`variable <variable>`.  In each case, the compute, fix, or variable
+must produce a global quantity, quantity, not a per-atom or local
+quantity.  If you wish to spatial- or time-average or histogram per-atom
+quantities from a compute, fix, or variable, then see the :doc:`fix
+ave/chunk <fix_ave_chunk>`, :doc:`fix ave/atom <fix_ave_atom>`, or
+:doc:`fix ave/histo <fix_ave_histo>` commands.  If you wish to sum a
+per-atom quantity into a single global quantity, see the :doc:`compute
+reduce <compute_reduce>` command.
+
+:doc:`Computes <compute>` that produce global quantities are those which
```

Comment:
This is too simplistic, there are also computes with "local" or "grid" in their name, and some of atom/grid/local also calculate global quantities.  I would just say many computes and fixes compute global quantities; see their doc pages for details.